vmm/Cargo.toml

```diff
@@ -15,11 +15,14 @@ log = "*"
 net_util = { path = "../net_util" }
 pci = {path = "../pci"}
 qcow = { path = "../qcow" }
-linux-loader = { git = "https://github.com/bjzhjing/linux-loader" }
 vm-virtio = { path = "../vm-virtio" }
 vm-allocator = { path = "../vm-allocator" }
 vmm-sys-util = { git = "https://github.com/rust-vmm/vmm-sys-util" }
 
+[dependencies.linux-loader]
+git = "https://github.com/bjzhjing/linux-loader"
+features = ["elf"]
+
```

Comment:
Yes, I had that fix locally but it bothers me. The linux-loader code used to build vmm should be coming from what .cargo/config points at:
```
[source."vendor+https://github.com/bjzhjing/linux-loader"]
directory = "./vendor/git-4af8f4552cd0d200"
```

So it should build against our vendored linux-loader, which does not need the elf feature.

Reply:
Hmm, I see that the linux-loader vendoring is also updated...